test/fungus_toast/games/grid_test.exs

```diff
@@ -0,0 +1,55 @@
+defmodule FungusToast.Games.GridTest do
+  use ExUnit.Case, async: true
+  alias FungusToast.Games.Grid
+
+  describe "create_starting_grid/3" do
+    test "that each player gets a new cell" do
+      new_grid = Grid.create_starting_grid(20, 20, [10, 20, 30])
+
+      assert Enum.count(new_grid) == 3
+    end
+
+    defp find_grid_cell_for_player(grid, player_id) do
+      tuple = Enum.find(grid, fn {_, v} -> v.player_id == player_id end)
+      elem(tuple, 1)
+    end
+
+    defp assert_index_between(grid, player_id, start_value_inclusive, end_value_exclusive) do
+      grid_cell = find_grid_cell_for_player(grid, player_id)
+      assert(grid_cell.index >= start_value_inclusive)
+      assert(grid_cell.index < end_value_exclusive)
```

Comment:
Why is this exclusive?

Reply:
It should be inclusive now that I changed the values to 0-19 instead of 0-20 -- good catch! I'll fix this now.